CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.11.1 (2018-01-08)
+
+A small release fixing a somewhat nasty bug involving running benchmarks in
+parallel.
```

Comment:
unsure if we can make that happen as `0.11.1` - saving and storing is already implemented and on master + only missing little things to go live. As this is no huge deal breaker I'd like to avoid the overhead of branching from 0.11.0 to have it clean and rather release `0.12.0` this week. Wdyt?

Reply:
Yeah, fine by me! Yay for more new features!